worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitAdapter.java

```diff
@@ -363,6 +357,7 @@ public static GameMode adapt(org.bukkit.GameMode gameMode) {
         return GameModes.get(gameMode.name().toLowerCase(Locale.ROOT));
     }
 
+    private static final Biome[] biomeValuesCache = Biome.values();
```

Comment:
What's the purpose of caching these? Given the lookup occurs within a cached call I can't see it being _that_ major in terms of performance, especially given there aren't tens of thousands of biomes